### PR TITLE
fix: fix sidebar href

### LINKED ISSUE #2994: [Refactor][config-ui] reload when click any nav item
## What and why to refactor
image
In react app, it should not reload when click any nav item.

## Describe the solution you'd like
No idea.

### PR DESCRIPTION
**Pull request by likyh**
# Summary

fix href. Now it will not reload when click.

### Does this close any open issues?
Closes #2994

### CODE REVIEW

File: config-ui/src/components/Sidebar/SidebarMenu.jsx [hunk 1]
```diff
@@ -18,13 +18,25 @@
...
 const SidebarMenu = (props) => {
+  const history = useHistory()
   const { menu } = props
   // const activeRoute = useRouteMatch()
 
   useEffect(() => {}, [menu])
 
+  const handleProviderClick = (route) => {
+    if (route) {
+      if (route.includes('//')) {
```
Comment (mintsweet) [inline config-ui/src/components/Sidebar/SidebarMenu.jsx:32]: why do you do this? **//** will bring any issues?

Comment (likyh) [reply config-ui/src/components/Sidebar/SidebarMenu.jsx:32]: URL contains `//` perhaps means it is an absolute URL. like `http://...../...`.  sometimes http(s) can be ellipsis to `//baidu.com/`.

File: config-ui/src/components/Sidebar/SidebarMenu.jsx [hunk 2]
```diff
@@ -18,13 +18,25 @@
...
+import { useHistory } from 'react-router-dom'
 
 const SidebarMenu = (props) => {
+  const history = useHistory()
   const { menu } = props
   // const activeRoute = useRouteMatch()
 
   useEffect(() => {}, [menu])
 
+  const handleProviderClick = (route) => {
```
[the remark this answers is not in the document]
Comment (likyh) [reply config-ui/src/components/Sidebar/SidebarMenu.jsx]: Although now there is no html tag with target, it is a good point. Fixed.

**Comment by e2corporation** [comment]
@likyh PR is failing **Lint Commit Message** CI Check

**Review by e2corporation — APPROVED** [review]
LGTM

**Pull request merged**